common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -42,20 +42,21 @@ div.video {
   .wrapper-downloads {
     margin: 0;
     padding: 0;
+    list-style: none;
```

Comment:
This doesn't work so just using the overrides here. Cleanup forthcoming in pattern library/cleanup.